subworkflows/local/qiime2_diversity.nf

```diff
@@ -52,8 +52,10 @@ workflow QIIME2_DIVERSITY {
 
         //adonis ( ch_metadata, DIVERSITY_CORE.out.qza )
         if (params.qiime_adonis_formula) {
+            ch_qiime_adonis_formula = Channel.from(params.qiime_adonis_formula.tokenize(','))
```

Comment:
```suggestion
            ch_qiime_adonis_formula = Channel.of(params.qiime_adonis_formula.tokenize(','))
```

Reply:
Thanks as well, .from seems to be deprecated, so I replace all by .of

Reply:
Applied it, see below